build.gradle

```diff
@@ -47,4 +48,20 @@ jar {
     manifest {
         attributes 'Main-Class': mainClassName
     }
-}
\ No newline at end of file
+}
+
+test {
+    ignoreFailures = true
```

Comment:
Is this a worrying parameter, is this going to affect any of your processes.

In my case we need it, otherwise a test failure means coverage information is not created

Reply:
Nothing to worry about.